wger/nutrition/sync.py

```diff
@@ -216,40 +221,73 @@ def download_ingredient_images(
 def sync_ingredients(
     print_fn,
     remote_url=settings.WGER_SETTINGS['WGER_INSTANCE'],
+    language_codes: Optional[str] = None,
     style_fn=lambda x: x,
 ):
     """Synchronize the ingredients from the remote server"""
+
+    def _sync_ingredients(language_id: Optional[int] = None):
+        if language_id is not None:
+            url = make_uri(
+                INGREDIENTS_ENDPOINT,
+                server_url=remote_url,
+                query={'limit': API_MAX_ITEMS, 'language': language_id},
```

Comment:
I'd just allow passing all languages at once, seems easier that way. For this we just need to add 'in' to `IngredientFilterSet`, like in 'id'. Then we can do `query={'limit': API_MAX_ITEMS, 'language__in': ','.join([languages])}` (this also works if we only pass one)